app/src/main/java/com/example/benchproject/app/movie/details/view/MovieDetailsFragment.kt

```diff
@@ -5,16 +5,27 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.navArgs
+import com.example.benchproject.app.movie.details.viewModel.MovieDetailsViewModel
 import com.example.benchproject.databinding.MovieDetailsFragmentBinding
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class MovieDetailsFragment : Fragment() {
 
+    private val movieDetailsViewModel: MovieDetailsViewModel by viewModels()
+    private val args: MovieDetailsFragmentArgs by navArgs()
```

Comment:
You could get args directly in ViewModel, there are not needed in Fragment. Read about `savedStateHandle` in Hilt ViewModel, args can be taken from it

Reply:
Ok, I will read about this